test/evergreen.yml

```diff
@@ -1501,6 +1501,69 @@ tasks:
+          posix_configure_flags: --enable-silent-rules --enable-diagnostic --enable-strict --enable-python --with-builtins=lz4,snappy,zlib
+      - func: "make check all"
+      - func: "unit test"
+        vars:
+          unit_test_args: -v 2 --long
+      - func: "test format"
+        vars:
+          test_format_args: -1 -q -c ../../../test/format/CONFIG.stress checkpoints=1 leak_memory=0 mmap=1 file_type=row compression=snappy logging=1 logging_compression=snappy logging_prealloc=1
+      - func: "test format"
+        vars:
+          test_format_args: -1 -q -c ../../../test/format/CONFIG.stress checkpoints=1 leak_memory=0 mmap=1 file_type=row alter=1 backups=1 compaction=1 data_extend=1 prepare=1 rebalance=1 salvage=1 statistics=1 statistics_server=1 verify=1
+      - func: "test format"
+        vars:
+          test_format_args: -1 -q -c ../../../test/format/CONFIG.stress checkpoints=1 leak_memory=0 mmap=1 file_type=row direct_io=1 firstfit=1 internal_key_truncation=1
+      - func: "test format"
+        vars:
+          test_format_args: -1 -q -c ../../../test/format/CONFIG.stress checkpoints=1 leak_memory=0 mmap=1 file_type=row checkpoints=0 in_memory=1 reverse=1 truncate=1
+      - func: "test format"
+        vars:
+          test_format_args: -1 -q -c ../../../test/format/CONFIG.stress checkpoints=1 leak_memory=0 mmap=1 file_type=row compression=zlib huffman_key=1 huffman_value=1
+      - func: "test format"
+        vars:
+          test_format_args: -1 -q -c ../../../test/format/CONFIG.stress checkpoints=1 leak_memory=0 mmap=1 file_type=row isolation=random transaction_timestamps=0
+      - func: "test format"
+        vars:
+          test_format_args: -1 -q -c ../../../test/format/CONFIG.stress checkpoints=1 leak_memory=0 mmap=1 file_type=row data_source=lsm bloom=1
+      - func: "test format"
+        vars:
+          test_format_args: -1 -q -c ../../../test/format/CONFIG.stress checkpoints=1 leak_memory=0 mmap=1 file_type=var compression=snappy checksum=uncompressed dictionary=1 repeat_data_pct=10
+      - func: "test format"
+        vars:
+          test_format_args: -1 -q -c ../../../test/format/CONFIG.stress checkpoints=1 leak_memory=0 mmap=1 file_type=row compression=lz4 prefix_compression=1 leaf_page_max=9 internal_page_max=9 key_min=256 value_min=256
+      - func: "test format"
+        vars:
+          test_format_args: -1 -q -c ../../../test/format/CONFIG.stress checkpoints=1 leak_memory=0 mmap=1 file_type=var leaf_page_max=9 internal_page_max=9 value_min=256
+      - func: "test format"
+        vars:
+          test_format_argas: -1 -q -c ../../../test/format/CONFIG.stress checkpoints=1 leak_memory=0 mmap=1 file_type=fix
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix"
+          script: |
+            set -o errexit
+            set -o verbose
+            pip install gcovr --user
```

Comment:
Add a `# FIX ME` comment mentioning this line should be removed once the BUILD ticket is resolved.